lmdeploy/pytorch/models/qwen2.py

```diff
@@ -133,6 +133,7 @@ def forward(
         past_key_value: Optional[Tuple[torch.Tensor]] = None,
         output_attentions: bool = False,
         use_cache: bool = False,
+        cache_position: Optional[torch.LongTensor] = None,
```

Comment:
as we will not use these args, just add a `**kwargs` should be ok.

Reply:
ok